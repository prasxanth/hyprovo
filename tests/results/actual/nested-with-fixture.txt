2021-11-24 17:25:51.893 - INFO - Pass ✓... (test-add-variable): add pass
2021-11-24 17:25:51.896 - ERROR - Fail ✗... (test-add-variable): add fail
2021-11-24 17:25:51.902 - INFO - Pass ✓... (test-mul-variable): mul pass
2021-11-24 17:25:51.905 - ERROR - Fail ✗... (test-mul-variable): mul fail
