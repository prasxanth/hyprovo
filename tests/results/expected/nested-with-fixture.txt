2021-11-24 17:24:55.472 - INFO - Pass ✓... (test-add-variable): add pass
2021-11-24 17:24:55.474 - ERROR - Fail ✗... (test-add-variable): add fail
2021-11-24 17:24:55.479 - INFO - Pass ✓... (test-mul-variable): mul pass
2021-11-24 17:24:55.481 - ERROR - Fail ✗... (test-mul-variable): mul fail
